internal/resources/audit_log_export.go

```diff
@@ -225,12 +225,9 @@ func (a *AuditLogExport) Update(_ context.Context, _ resource.UpdateRequest, res
 	)
 }
 
-// Delete is not supported as audit log export API does not have delete endpoint.
-func (a *AuditLogExport) Delete(_ context.Context, _ resource.DeleteRequest, resp *resource.DeleteResponse) {
-	resp.Diagnostics.AddError(
-		"Audit Log Export does not support delete",
-		"Audit Log Export does not support delete",
-	)
+// Delete is not supported as audit log export API does not have delete endpoint, so we do a noop.
+func (a *AuditLogExport) Delete(ctx context.Context, _ resource.DeleteRequest, resp *resource.DeleteResponse) {
+	resp.State.RemoveResource(ctx)
```

Comment:
Thank you. That is correct, but I recall that we learned some time ago that if we leave the Delete block empty, the behavior remains the same. If you have time, could you please verify this possibility as well? Otherwise, it is fine if you are occupied; we can check later.

Reply:
Good point.  It does mentions this in documentation:
https://developer.hashicorp.com/terraform/plugin/framework/resources/delete#recommendations

I also validated in local testing.

Reply:
Addressed in 8d740a57008e7d442af26c310b450f38e7b2facd